spec/Section 3 -- Type System.md

```diff
@@ -2067,3 +2067,114 @@ to the relevant IETF specification.
 ```graphql example
 scalar UUID @specifiedBy(url: "https://tools.ietf.org/html/rfc4122")
 ```
+
+## Schema Coordinates
+
+SchemaCoordinate :
+  - Name
+  - Name . Name
+  - Name . Name ( Name : )
+  - @ Name
+  - @ Name ( Name : )
+
+:: A *schema coordinate* is a human readable string that uniquely identifies a
+*schema element* within a GraphQL Schema.
+
+:: A *schema element* is a specific instance of a named type, type field,
+input field, enum value, field argument, directive, or directive argument.
+
+A *schema coordinate* is always unique. Each *schema element* may be referenced
+by exactly one possible schema coordinate.
+
+A *schema coordinate* may refer to either a defined or built-in *schema element*.
+For example, `String` and `@deprecated(reason:)` are both valid schema
+coordinates which refer to built-in schema elements.
+
+Note: A {SchemaCoordinate} is not a definition within a GraphQL {Document}, but
+a separate stand-alone grammar, intended to be used by tools to reference types,
```

Comment:
I don't think it's necessary to hyphenate `stand-alone` these days:

```suggestion
a separate standalone grammar, intended to be used by tools to reference types,
```

Reply:
TIL https://brians.wsu.edu/2016/05/31/standalone-stand-alone/